tests/infra/member.py

```diff
@@ -148,7 +148,7 @@ def ack(self, remote_node):
         state_digest = self.update_ack_state_digest(remote_node)
         with remote_node.client(*self.auth(write=True)) as mc:
             r = mc.post("/gov/ack", body=state_digest)
-            assert r.status_code == 200, f"Error ACK: {r}"
+            assert r.status_code == 204, f"Error ACK: {r}"
```

Comment:
We have a mix of int literals and `http.HTTPStatus` values in our Python, but I think we should prefer the latter?

```suggestion
            assert r.status_code == http.HTTPStatus.NO_CONTENT, f"Error ACK: {r}"
```